### PR TITLE
routesのレスポンスバリデーション

### PR DESCRIPTION
**Pull request by takoring**

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by wing328** [comment]
Looks like it's not a breaking change so it should target master instead.

File: modules/openapi-generator/src/main/resources/Javascript/partial_model_generic.mustache [hunk 1]
```diff
@@ -77,6 +77,13 @@
       {{/parentModel}}
       {{/useInheritance}}
       {{#vars}}
+      {{#requiredValidate}}
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/Javascript/partial_model_generic.mustache]: I don't see any reference to `requiredValidate` in the code base. Where does this tag come from?

File: modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm/infrastructure/ApiClient.kt.mustache [hunk 2]
```diff
@@ -21,6 +21,7 @@ import okhttp3.ResponseBody
 import okhttp3.MediaType.Companion.toMediaTypeOrNull
 {{/jvm-okhttp4}}
 import okhttp3.Request
+import okhttp3.Credentials
```
Comment (4brunu) [inline modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm/infrastructure/ApiClient.kt.mustache]: Why is it adding this import?

File: modules/openapi-generator/src/main/resources/Javascript/partial_model_generic.mustache (hunk 1 above)
Comment (yoshimotocamobile) [reply modules/openapi-generator/src/main/resources/Javascript/partial_model_generic.mustache]: additional-properties requiredValidate implementation addition

**Comment by yoshimotocamobile** [comment]
> Looks like it's not a breaking change so it should target master instead.

I use model for a response of backend.
As it becomes the analysis error when required does not enter in sdk of kotlin-client or swift-client, I do validation at the time of a response in a server side

File: modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm/infrastructure/ApiClient.kt.mustache (hunk 2 above)
Comment (4brunu) [inline modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm/infrastructure/ApiClient.kt.mustache]: @yoshimotocamobile Thanks for your contribution 🙂 
Since https://github.com/OpenAPITools/openapi-generator/pull/4772 was merged and already fixes this, could you please revert this change? Thanks
